util/os-runner/Makefile

```diff
@@ -0,0 +1,26 @@
+OS_HOME=$$HOME/school/os
```

Comment:
Tell the students to change this variable to their own path in a `README.md` file.